### PR TITLE
[INTEL MKL] MKL DNN 0.x code clean -- common_runtime and ops modules

### PR DESCRIPTION
**Pull request by gzmkl**
DNN 0.x cleanup of common_runtime and ops modules

(1) Remove all DNN 0.x related code;

(2) low-hanging-fruit - remove some non-production code (commented-out statements or unnecessary comments)

### CODE REVIEW

File: tensorflow/core/common_runtime/mkl_layout_pass_test.cc [hunk 1]
```diff
@@ -5166,7 +5159,6 @@ static void BM_MklLayoutRewritePass(int iters, int op_nodes) {
     }
     iters -= N;  // Our benchmark units are individual graph nodes,
                  // not whole graphs
-    // delete graph;
```
Comment (penpornk) [inline tensorflow/core/common_runtime/mkl_layout_pass_test.cc:5169]: Shouldn't this be uncommented instead?

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for the clean-up and I'm so sorry for the delay!

File: tensorflow/core/common_runtime/mkl_layout_pass_test.cc (hunk 1 above)
Comment (gzmkl) [reply tensorflow/core/common_runtime/mkl_layout_pass_test.cc:5169]: yes, I will do

Comment (gzmkl) [reply tensorflow/core/common_runtime/mkl_layout_pass_test.cc:5169]: @penpornk  I double-checked and found that this is a commented out statement, which should be deleted.

Comment (penpornk) [reply tensorflow/core/common_runtime/mkl_layout_pass_test.cc:5169]: Thank you for double-checking! But won't we have memory leaks without this line (`graph` gets reassigned to a new address in line 5154 in each iteration). 

Maybe change line 5154 to 
```c++
std::unique_ptr<Graph> graph(new Graph(OpRegistry::Global()));
``` 
instead?

Comment (gzmkl) [reply tensorflow/core/common_runtime/mkl_layout_pass_test.cc:5169]: Good suggestion, I will take a look. 

The current implementation may have mem leaky issue, which has not been detected due to being test only :)

Comment (gzmkl) [reply tensorflow/core/common_runtime/mkl_layout_pass_test.cc:5169]: Done with code change (per suggestion of using unique_ptr) and with successful local test!

**Review by penpornk — APPROVED** [review]
Thank you for the changes! :)

**Pull request merged**